sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -342,7 +340,6 @@ gnrc_rpl_instance_t *gnrc_rpl_root_instance_init(uint8_t instance_id, ipv6_addr_
     gnrc_netif_t *netif;
     gnrc_rpl_instance_t *inst = NULL;
     gnrc_rpl_dodag_t *dodag = NULL;
-    kernel_pid_t iface;
```

Comment:
Unrelated, but I figured this variable is not really needed. Hope it's okay I piggy-backed that.

Reply:
It's okay, I don't mind the small piggy backing cleanups in these PRs. I prefer a separate commit though, but not enough to really start nagging about it.

Reply:
Done anyway ;-)